plsql-frontend/src/main/java/org/sonar/plugins/plsqlopen/api/PlSqlKeyword.java

```diff
@@ -87,6 +88,7 @@
     OVERLAPS("overlaps", true),
     PROCEDURE("procedure", true),
     PUBLIC("public", true),
+    RENAME("rename", true),
```

Comment:
"rename" is not a reserved keyword too. Example: [ftp_util_pkg.**rename**](https://github.com/mortenbra/alexandria-plsql-utils/blob/bc33845a33b0d31f508e759283acb00c28dbf1a7/ora/ftp_util_pkg.pks#L104) from the alexandria-plsql-utils.